lib/std/math.zig

```diff
@@ -335,7 +370,7 @@ pub inline fn exp2(value: anytype) @TypeOf(value) {
     return @exp2(value);
 }
 
-pub const complex = @import("math/complex.zig");
+const complex = @import("math/complex.zig");
```

Comment:
This seems likely to break a bunch of code. Im not sure that its necessary to hide the `complex` namespace